packages/combobox/src/index.tsx

```diff
@@ -782,6 +827,13 @@ export type ComboboxOptionProps = {
    * @see Docs https://reacttraining.com/reach-ui/combobox#comboboxoption-value
    */
   value: string;
+  /**
+   * Custom data that will be passed to the `onSelect` of the `Combobox` as a
+   * second argument.
+   *
+   * @see Docs https://reacttraining.com/reach-ui/combobox#comboboxoption-data
```

Comment:
@CodingDive looks like this anchor section is missing from the [website source page][1]. Should we update that section of the website to reflect your changes?

[1]: https://github.com/reach/reach-ui/blob/1a17f732b70a61ad57576d149312265432d8dd30/website/src/pages/combobox.mdx#comboboxoption

Reply:
Yes, will do 😊